staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/deepcopy.go

```diff
@@ -26,11 +26,13 @@ func (in *JSONSchemaProps) DeepCopy() *JSONSchemaProps {
 	*out = *in
 
 	if in.Default != nil {
-		out.Default = deepCopyJSON(in.Default)
+		defaultJSON := JSON(deepCopyJSON(*(in.Default)))
```

Comment:
you might want to set nil if in.Default is nil.

Reply:
right :+1: